app/agents.py

```
@@ -0,0 +1,86 @@
+from sqlalchemy import text
+from sqlalchemy.orm import Session
+from db.database import SessionLocal
+
+def directory_db_query(query_string):
+    print("executing db query for directory db: " + query_string)
+    results = []
+    db = SessionLocal()
+    try:
+        my_query = text(query_string)
+        results = db.execute(my_query).fetchall()
+    except Exception as e:
```

Ruff (0.12.2)

Do not catch blind exception: `Exception`

(BLE001)